ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -167,6 +167,17 @@ protected virtual ReactRootView CreateRootView()
         }
 
         /// <summary>
+        /// Called during OnCreate, to set the RootView's background
+        /// </summary>
+        /// <remarks>
+        /// Subclasses may override this method if it needs to use a custom background.
+        /// </remarks>
+        protected virtual void SetRootViewBackground()
+        {
```

Comment:
Yep, can do that. How would you override the setter though? The problem is that there is currently no way to override this behavior in `onCreate`, so I'm not sure how to solve it with a setter.